allennlp/commands/predict.py

```diff
@@ -56,7 +56,8 @@
         'bidaf': 'machine-comprehension',
         'simple_tagger': 'sentence-tagger',
         'crf_tagger': 'sentence-tagger',
-        'coref': 'coreference-resolution'
+        'coref': 'coreference-resolution',
+        'constituency_parser': 'constituency-parser'
```

Comment:
I'd leave a comma at the end of this line - it's valid python, and it makes maintaining it easier.